modin/pandas/series.py

```diff
@@ -308,22 +370,39 @@ def __rtruediv__(self, left):
     __itruediv__ = __truediv__
 
     @property
-    def values(self):
+    def values(self):  # noqa: RT01, D200
+        """
+        Return Series as ndarray or ndarray-like depending on the dtype.
+        """
         return super(Series, self).to_numpy().flatten()
 
-    def add(self, other, level=None, fill_value=None, axis=0):
+    def add(self, other, level=None, fill_value=None, axis=0):  # noqa: PR01, RT01, D200
```

Comment:
isn't the documentation for `.add` and other public API methods inherited from `pandas.Series` by `_inherit_docstrings` decorator?..

Reply:
Right, docstrings for all `Series` methods can be inherited from `pandas.Series`, but in this case these docstrings can have multiple `pandas` related stuff like referring to `pandas` methods, notes about changes relative to `pandas` version, etc. (see [example](https://pandas.pydata.org/docs/reference/api/pandas.Series.factorize.html)). To avoid such issues docstrings were composed from short summary and link to `pandas` API, example for `Series.add`:
```
        Return Addition of series and other, element-wise (binary operator add).

        Notes
        -----
        See `pandas API documentation for pandas.Series.add <https://pandas.pydata.org/pandas-docs/version/1.2.3/reference/api/pandas.Series.add.html>`_ for more.
```
So, what do you think - can we leave this approach? Alternatively we can use unchanged `pandas` docstrings or parse original `pandas` docstrings manually.

Reply:
I don't have a preference, but I feel that having less code while retaining the docs should be better overall - less stuff to maintain.